### PR TITLE
Remove Shard Committees 3 - Get RANDAO Mix Helper

### PR DESCRIPTION
**Pull request by terenc3t**
This is part of #1352

This PR implements and tests the following get_randao_mix helper function. `get_randao_mix` is used to process validatory registry

```
def get_randao_mix(state: BeaconState,
                   slot: int) -> Bytes32:
    """
    Returns the randao mix at a recent ``slot``.
    """
    assert state.slot < slot + LATEST_RANDAO_MIXES_LENGTH
    assert slot <= state.slot
    return state.latest_randao_mixes[slot % LATEST_RANDAO_MIXES_LENGTH]
```

### CODE REVIEW

File: beacon-chain/core/epoch/epoch_operations_test.go [hunk 1]
```diff
@@ -675,3 +669,51 @@ func TestInclusionDistance_NotFound(t *testing.T) {
 		t.Fatal("InclusionDistance should have failed")
 	}
 }
+
+func TestRandaoMix_Ok(t *testing.T) {
+	randaoMixes := make([][]byte, config.LatestRandaoMixesLength)
```
Comment (nisdas) [inline beacon-chain/core/epoch/epoch_operations_test.go]: where does config come from ?

File: beacon-chain/core/epoch/epoch_operations_test.go [hunk 2]
```diff
@@ -675,3 +669,51 @@ func TestInclusionDistance_NotFound(t *testing.T) {
...
+	}
+}
+
+func TestRandaoMix_OutOfBound(t *testing.T) {
+	want := fmt.Sprintf(
+		"input randaoMix slot %d out of bounds: %d <= slot < %d",
+		100, 0, 0,
+	)
+	if _, err := randaoMix(&pb.BeaconState{}, 100); !strings.Contains(err.Error(), want) {
+		t.Errorf("Expected: %s, received: %v", want, err)
```
Comment (nisdas) [inline beacon-chain/core/epoch/epoch_operations_test.go]: make this `err.Error()` and change the formatting to `%s`

File: beacon-chain/core/epoch/epoch_operations_test.go (hunk 1 above)
Comment (terenc3t) [reply beacon-chain/core/epoch/epoch_operations_test.go]: defined on top of `epoch_processing.go`:

`var config = params.BeaconConfig()`

makes everything looks a lot neater w/o the whole `params.BeaconConfig()`

Comment (nisdas) [reply beacon-chain/core/epoch/epoch_operations_test.go]: Ah , thanks. Didnt see that

**Pull request merged**